src/extensions/boosts.py

```diff
@@ -26,7 +26,7 @@ def build_boost_message(
 ) -> str:
     assert message_type in BOOST_MESSAGE_TYPES
 
-    base_message = f"{booster_user.display_name} just boosted the server"
+    base_message = f"<@{booster_user.id}> just boosted the server 🎉"
```

Comment:
if the multiple boosts message is appended to this, the 🎉 will appear before it. This can instead be added after the "!" below